code/__DEFINES/tgs.config.dm

```diff
@@ -1,5 +1,5 @@
 #define TGS_EXTERNAL_CONFIGURATION
-#define TGS_V3_API
+//#define TGS_V3_API

```

Comment:
Seems to work, instancing didn't happen on two seperate reboots and data now works.